plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -79,4 +88,54 @@ public void testShowCreateTable()
                         format("   location = '.*/" + schemaName + "/region'\n") +
                         "\\)");
     }
+
+    @Test
+    public void testListInformationSchemaMetadataNotFound()
```

Comment:
A detailed tests belongs more to `BaseIcebergConnectorTest` rather than `BaseIcebergConnectorSmokeTest`
any reason this is here?

Reply:
Actually, I was adding to `BaseIcebergConnectorSmokeTest` at first, but test doesn't have coverages for S3 nor minio.    Since #11764 depends on S3, `BaseIcebergConnectorTest` makes sense to me.